### PR TITLE
Adding documentation

### PR DESCRIPTION
**Pull request by typiconman**
@pgmmpk: Please check. This is my idea on how to create documentation for fonts-churchslavonic TeX package. I propose to put all of the fonts in the same document. Also, I think we can get rid of the codecharts, as they take up a lot of space. What do you think?
If it looks OK, I will finish the doc and add the Makefile.

### CODE REVIEW

**Comment by pgmmpk** [comment]
@typiconman this looks great. Actually, the doc is very detailed, possibly too detailed. But this is good
since this is not just TeX docs, but rather fonts-cu docs.

**Comment by typiconman** [comment]
Merging.

File: README.ctan [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	tlmgr install fonts-churchslavonic
+
+
+Using fonts
+-----------
+
+Use "fontspec" LaTeX package to load any of the installed fonts. For example:
+
+	\newfontfamily\ponomar[Script=Cyrillic,HyphenChar=_]{Ponomar Unicode}
```
Comment (pgmmpk) [inline README.ctan]: Fonts installed by tlmgr can not be loaded by name (unless also installed on system level). Lets change this to "PonomarUnicode.otf".

Comment (typiconman) [inline README.ctan]: Fixed this in https://github.com/typiconman/fonts-cu/pull/13

**Pull request merged**